java/Oleksii-Riabov/src/com/zoolatech/lecture4/tasks/_4/Weight.java

```diff
@@ -0,0 +1,86 @@
+package com.zoolatech.lecture4.tasks._4;
+
+public enum Weight {
+    GRAM ("g"){
+        @Override
+        public double convert(double d, Weight weight) {
+            switch (weight) {
+                case GRAM -> {return d;}
```

Comment:
braces for the return statement can be omitted

Reply:
Switch blocks has been changed on switch expressions, return statement has been removed.